app/src/main/res/layout/view_route_mode.xml

```diff
@@ -1,11 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
 
     <android.support.v4.view.ViewPager
         android:id="@+id/instruction_pager"
         android:layout_width="match_parent"
-        android:layout_height="75dp" />
+        android:layout_height="@dimen/pager_height"/>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_gravity="bottom"
+        android:layout_alignParentBottom="true">
+
+        <LinearLayout
```

Comment:
Do we need a LinearLayout inside a RelativeLayout here? Can these be collapsed into a single container for the footer?